src/azure.js

```diff
@@ -82,3 +82,86 @@ api.declare({
     expiry:   expiry.toJSON()
   });
 });
+
+api.declare({
+  method:     'get',
+  route:      '/azure/:account/blob/:container/:level',
+  name:       'azureBlobSAS',
+  input:      undefined,
+  output:     'azure-blob-response.json#',
+  deferAuth:  true,
+  stability:  'stable',
+  scopes:     [['auth:azure-blob:<level>:<account>/<container>']],
+  title:      "Get Shared-Access-Signature for Azure Blob",
+  description: [
+    "Get a shared access signature (SAS) string for use with a specific Azure",
+    "Blob Storage container. If the level is read-write, the container will be created, " +
+    "if it doesn't already exists."
+  ].join('\n')
+}, async function(req, res){
+  // Get parameters
+  let account = req.params.account;
+  let containerName = req.params.container;
+  let level = req.params.level;
+
+  if (['read-write', 'read-only'].indexOf(level) < 0) {
+    return res.reportError('InvalidRequestArguments',
+      `Level '${level}' is not valid. Must be one of ['read-write', 'read-only'].`);
+  }
+
+  // Check that the client is authorized to access given account and container
+  if (!req.satisfies({
+      account:      account,
+      container:    containerName,
+      level:        level,
+    })) {
```

Comment:
Please use the short hand: `req.satisfies({account, container: containerName, level})`
If you changed `containerName` to `container` it would work even better :)